core-bundle/src/EventListener/HttpCache/StripCookiesSubscriber.php

```diff
@@ -82,7 +94,7 @@ public function preHandle(CacheEvent $event): void
         if (0 !== \count($this->whitelist)) {
             $this->filterCookies($request, $this->whitelist, true);
         } else {
-            $this->filterCookies($request, self::BLACKLIST);
+            $this->filterCookies($request, array_diff(self::BLACKLIST, $this->disabledFromBlacklist));
```

Comment:
Using array_diff here could lead to unexpected results I think. If someone wants to allow the `_utma` cookie and calls `disableFromBlacklist(['_utma'])` the cookie would still get stripped because the regex for this cookie is `__utm.+`.

I think we should extend the `filterCookies()` method and remove every name from `$removeCookies` that matches the `disableFromBlacklist` array.

Reply:
I actually wanted to explicitly disallow this use case. If you want to remove something from the blacklist, specify it correctly. You can still use the whitelist if you want to have full control.

Reply:
But what if we change one of the regexes in the blacklist in a future update of Contao? The `disableFromBlacklist()` would no longer work than for that regex and this would be BC break.

Reply:
I thought about that, would be easy to solve with an alias in the `disableFromBlacklist()` method 😉

Reply:
But what should I do then if I want to remove just the `_utma` (not all `_utm*`) cookie from the blacklist?

Reply:
That's not a use case imho. Either you need all of the `_utm` query parameters or none. 
But you're free trying to optimize this in a later version.

Reply:
I could also think of something like a `bimodal_transport` cookie which is currently blacklisted by `(.*)?modal(.*)?`. But you are right, I will provide a separate pull request for this 👍